docs/src/components/navbar.tsx

```diff
@@ -0,0 +1,83 @@
+import { useMemo, useState } from "react";
+
+import { buttonVariants } from "@/components/ui/button";
+import { Input } from "@/components/ui/input";
+import { ScrollArea } from "@/components/ui/scroll-area";
+import { FunctionTag } from "@/components/function-tag";
+import { cn } from "@/lib/utils";
+
+import { toPairs } from "../../../src";
```

Comment:
You can't do this, this doesn't compile because the path is outside of the typescript project. Add remeda to the project and use it as an external dependency.
```suggestion
import { toPairs } from "remeda";
```